src/main/java/httpserver/http/route/RouteHandler.java

```diff
@@ -18,35 +20,20 @@ public String getResponse() {
         var requestSplitter = new RequestSplitter(request);
         var requestMethod = requestSplitter.getRequestMethod();
         var requestPath = requestSplitter.getRequestPath();
+        return selectResponse(requestMethod, requestPath);
+    }
 
-        if (requestMethod.equals(RequestMethod.GET.toString())
-                && requestPath.equals("/simple_get")) {
-            return simpleGetResponse();
-        } else if (requestMethod.equals(RequestMethod.OPTIONS.toString())
-                && requestPath.equals("/method_options")) {
-            return methodOptionsResponse();
+    private String selectResponse(String requestMethod, String requestPath) {
```

Comment:
See above - could just accept a `Request` object, if it's using all the methods on `Request`.